### DISCUSSION TITLE
getting 401 unauthorized at /users/me

### DISCUSSION THREAD

**Discussion by ranaharoon3222** [directus/directus#11392] [Archive]
am getting 401 unauthorized at /users/me

`{
  "errors": [
    {
      "message": "Invalid user credentials.",
      "extensions": {
        "code": "INVALID_CREDENTIALS"
      }
    }
  ]
}`

### To Reproduce

just hit  /users/me   you will get error

### Errors Shown

`{
  "errors": [
    {
      "message": "Invalid user credentials.",
      "extensions": {
        "code": "INVALID_CREDENTIALS"
      }
    }
  ]
}`

**Comment by rijkvanzanten** [comment]
Did you authenticate?

**Comment by ranaharoon3222** [comment]
yes of course i authenticated as admin

**Reply by rijkvanzanten** [reply]
How did you authenticate. 

> yes of course


For the record, that's absolutely not obvious based on your original message.

**Comment by ranaharoon3222** [comment]
i authenticate as admin using app just simple process

https://www.loom.com/share/2bd7b0d8e4f2467c93ea20b1e459cdb9

**Reply by rijkvanzanten** [reply]
So you're not actually authenticating then. When you open that URL in a separate tab, you're not logged in. You have to provide an access token. See https://docs.directus.io/reference/authentication/#authentication for more information.

**Comment by ranaharoon3222** [comment]
hmm thanks understands it
